contextual-identities/manifest.json

```diff
@@ -0,0 +1,19 @@
+{
+  "browser_action": {
+    "browser_style": true,
+    "default_title": "Contextual Identities",
+    "default_popup": "context.html"
+  },
+  "homepage_url": "https://github.com/mdn/webextensions-examples/tree/master/contextual-identities",
+  "manifest_version": 2,
+  "name": "Contextual Identities",
+  "version": "1.0",
+  "background": {
+      "scripts": ["background.js"]
+  },
+  "permissions": [
+      "contextualIdentities",
+      "cookies",
+      "tabs"
+  ]
+}
```

Comment:
I know it's kind of a pain, but it would be great to have an icon for the browsr action. It looks unfinished otherwise.